requirements/docs.txt

```diff
@@ -0,0 +1,10 @@
+# these are dependencies required to build package documentaiton
+# ought to mirror 'docs' under options.extras_require in config.cfg
+-r extras.txt
+numpydoc
+pillow
+sphinx <= 2.4.4
```

Comment:
```suggestion
sphinx < 3.0
```
I'm not sure if we would want to do this instead on the chance that there's another bugfix release to 2.4.

Reply:
I think they're officially on the `3.0.x` path. Just today they released `3.0.2` and hopefully the fix to `sphinx-automodapi` will be merged soon.